tools/ci_build/build.py

```diff
@@ -589,6 +651,8 @@ def build_protoc_for_windows_host(cmake_path, source_dir, build_dir):
         raise BuildError("Couldn't build protoc.exe for host. Failing build.")
 
 def main():
+    update_version()
```

Comment:
Usually, build script won't modify any source file.  If you need a more convenient way to update all these files, could you please put this code into a separated script file? Because in 99% cases, most devs won't touch the version files.

Reply:
Good point, moved to separate file.